draft-ietf-quic-recovery.md

```diff
@@ -965,8 +964,8 @@ losses or increases in the ECN-CE counter.
 ## Tail Loss Probe
 
 A TLP packet MUST NOT be blocked by the sender's congestion controller. The
-sender MUST however count these bytes as additional bytes-in-flight, since a TLP
-adds network load without establishing packet loss.
+sender MUST however count TLP packets against bytes-in-flight, since a TLP adds
```

Comment:
inconsistent use of bytes-in-flight vs bytes_in_flight - technically one might be for text the other for pseudo code.

Reply:
Good point, how about we remove the - here, because it's spelled "bytes in flight" everywhere else.